lib/workers/repository/onboarding/pr/config-description.spec.ts

```diff
@@ -82,5 +82,15 @@ describe('workers/repository/onboarding/pr/config-description', () => {
       expect(res).toMatchSnapshot();
       expect(res.indexOf('`renovate.json`')).not.toBe(-1);
     });
+
+    it('include button selection if onboardingRebaseCheckbox is set to true', () => {
+      delete config.description;
+      config.schedule = ['before 5am'];
+      config.onboardingConfigFileName = '.github/renovate.json';
+      config.onboardingRebaseCheckbox = true;
+      const res = getConfigDesc(config);
+      expect(res).toMatchSnapshot();
+      expect(res.indexOf('Retry/Rebase')).not.toBe(-1);
```

Comment:
Not necessary since snapshot check, does the same.

Reply:
Removed the line